horreum-web/src/domain/alerting/PanelChart.tsx

```diff
@@ -36,7 +42,22 @@ function formatValue(value: number | string) {
         value /= 1000
         suffix = " k"
     }
-    return Number(value).toFixed(2) + suffix
+
+    value = Number(value)
+
+    const [_, decPart] = valueAsString.split(".")
+    let nonZeroIdx = 0
+    if (decPart) {
+        // set a limit to 10 decimal digits
+        for (nonZeroIdx; nonZeroIdx < 10; nonZeroIdx++) {
+            if (decPart[nonZeroIdx] != "0") {
+                // exit at the first non-zero digit
+                break;
+            }
+        }
+    }
```

Comment:
TBH I don't know if there is a better way to do it 🫤

Reply:
Me neither, but it should be fine I think.